README.md

```diff
@@ -16,27 +16,89 @@ Add `"extends": "stylelint-config-skyscanner"` to your `.stylelintrc`.
 
 ## Styleguide
 
-#### Patterns
+### Prefixes
+Don't prefix your CSS, write standards compliant code and let [`autoprefixer`](https://github.com/postcss/autoprefixer) automatically make it cross-browser compatible. Even better, `autoprefixer` and a config aligned to current browser usage comes bundled with `backpack-react-scripts` so you don't have to worry about it.
+Rules: `value-no-vendor-prefix`, `at-rule-no-vendor-prefix`, `media-feature-name-no-vendor-prefix`, `property-no-vendor-prefix`, `selector-no-vendor-prefix`
+
+### Selectors
+CSS is global by default. Whenever you write a CSS selector and expose it on a page, it's immediately in competition with every other selector on that page. This means that every selector has the potential to have unintended side effects by targeting unwanted elements or clashing with other selectors already defined.
+
+Use CSS modules to avoid conflicts between your selectors and any others on the page. CSS module supports comes with `backpack-react-scripts`.
```

Comment:
BEM is only for classes, not for all selectors though, which is I thought it should go in "Classes" and not in "Selectors". How strongly do you feel about this?